Cargo.toml

```diff
@@ -45,9 +48,9 @@ doc-comment = "0.3"
 kstring = { version = "0.1.0", path = "crates/kstring" }
 liquid-core = { version = "^0.19", path = "crates/core" }
 liquid-derive = { version = "^0.19", path = "crates/derive" }
+liquid-lib = { version = "^0.19", path = "crates/lib", optional = true }
 
 serde = { version = "1.0", optional = true, features = ["derive"] }
-deunicode = "1.0.0"
```

Comment:
url and unicode-segmentation should also be available for removal

Reply:
Looks like once_cell is only used by the library as well

Reply:
Even chrono, itertools.
regex is only used in tests, so moved to dev-dep